### PR TITLE
docs: added hint to debug via JavaScript Debug Terminal

### PR DESCRIPTION
**Pull request by IKoshelev**
Added description and screenshot how to debug Vitest tests with JavaScript Debug Terminal

### CODE REVIEW

File: docs/guide/debugging.md [hunk 1]
```diff
@@ -21,7 +21,11 @@ ndb npm run test
 
 ## VSCode
 
-To debug a test file in VSCode, create the following launch configuration.
+Quick way to debug tests in VSCode is via `JavaScript Debug Terminal`. Open a new `JavaScript Debug Terminal` and run `npm run test` or `vitest` directly. *this works with any code ran in Node, so will work with most JS testing frameworks*
+
+![image](https://user-images.githubusercontent.com/5594348/212169143-72bf39ce-f763-48f5-822a-0c8b2e6a8484.png)
```
Comment (sheremet-va) [inline docs/guide/debugging.md:26]: We shouldn't rely on outside sites for assets. Let's put it in the codebase.

**Review by sheremet-va — CHANGES_REQUESTED** [review]

File: docs/guide/debugging.md (hunk 1 above)
Comment (IKoshelev) [reply docs/guide/debugging.md:26]: `user-images.githubusercontent.com` is the site to which an image gets uploaded, if you simply drag it into your markdown in githubs online editor. AFAICS, this site hosts the logo from [main readme.md](https://raw.githubusercontent.com/vitest-dev/vitest/main/README.md) :

```
<img src="https://user-images.githubusercontent.com/11247099/145112184-a9ff6727-661c-439d-9ada-963124a281f7.png" height="150">
```

If the image should still go into a folder inside this repo - please provide link to the folder.

Comment (IKoshelev) [reply docs/guide/debugging.md:26]: @sheremet-va can you please comment on the question above?

Comment (sheremet-va) [reply docs/guide/debugging.md:26]: It's fine then. We cache this domain

**Pull request merged**